src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -277,20 +329,21 @@ private synchronized void updateNavigatorField()
      */
     private void initTasks()
     {
-        this.tasks.addTask(0, new EntityAISwimming(this));
+        int priority = 0;
+        this.tasks.addTask(priority, new EntityAISwimming(this));
 
         if (this.getColonyJob() == null || !"com.minecolonies.coremod.job.Guard".equals(this.getColonyJob().getName()))
         {
-            this.tasks.addTask(1, new EntityAICitizenAvoidEntity(this, EntityMob.class, DISTANCE_OF_ENTITY_AVOID, LATER_RUN_SPEED_AVOID, INITIAL_RUN_SPEED_AVOID));
+            this.tasks.addTask(++priority, new EntityAICitizenAvoidEntity(this, EntityMob.class, DISTANCE_OF_ENTITY_AVOID, LATER_RUN_SPEED_AVOID, INITIAL_RUN_SPEED_AVOID));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Extract this increment or decrement operator into a dedicated statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)